### PR TITLE
When I use the bbb server I need many docments on the create meeting.

### PR DESCRIPTION
**Pull request by dingyuanyang**
boolean current = "true".equals(document.@current.toString);
When I use the bbb server I need many docments on the create meeting.
So maybe we should have a flag for the default presentation

### CODE REVIEW

**Comment by zhu** [comment]
I thing it's not a good idea to have a current flag. The api caller may set no current flag or multi current flags, then the system will fail.

File: bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy [hunk 1]
```diff
@@ -1231,1076 +1549,602 @@ class ApiController {
...
-    }
+    def uploadDocuments(conf) {
+        log.debug("modify on 2018 09 10 ApiController#uploadDocuments(${conf.getInternalId()})");
 
-    ArrayList<String> recordIdList = new ArrayList<String>();
-    if (!StringUtils.isEmpty(recordId)) {
-      recordIdList=paramsProcessorUtil.decodeIds(recordId);
+        String requestBody = request.inputStream == null ? null : request.inputStream.text;
+        requestBody = StringUtils.isEmpty(requestBody) ? null : requestBody;
+        String preName = "中软大讲堂.pdf";
```
Comment (zhu) [inline bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy]: This is not a general default file name

Comment (dingyuanyang) [reply bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy]: It's OK ,we can change to "default.pdf"

Comment (dingyuanyang) [reply bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy]: but when a few documents be uploaded the system already die.so maybe you can change the code,make them batter.

**Comment by antobinary** [comment]
Can this pull request be closed now?

**Comment by dingyuanyang** [comment]
> Can this pull request be closed now?

Okey！

**Pull request closed without merging**